src/ValiDoc/Core/ValiDoc.cs

```diff
@@ -0,0 +1,73 @@
+﻿using FluentValidation;
+using FluentValidation.Internal;
+using FluentValidation.Validators;
+using System;
+using System.Collections.Generic;
+using ValiDoc.Output;
+
+namespace ValiDoc
+{
+    //TODO: This class is huge - should investigate splitting out dependencies
```

Comment:
👍 

Using DI will make this much cleaner at the expense of requiring the consumer to set it up.

Thankfully with .NET Core this can be achieved with extension methods to make it a one liner.